src/components/StatusBadge/StatusBadge.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+
+export interface StatusBadgeProps {
```

Comment:
Should we make some of these props required, i.e. `statusBadgeText` and `level` ? Would you have a badge without a level or text?

Reply:
I don't think `statusBadgeText` will be a prop since it will instead be children. I agree that `level` makes sense as a required prop, though—or we should have it default to say, `low`.

Reply:
In "StatusBadge.tsx", it's already defaulting to "low" and if `level` is passed it will use that value.

Reply:
Perfect, you've already anticipated us 😅 thanks Milos.